### ISSUE TITLE
Force interactive session on empty command

### ISSUE THREAD

**Issue by snk33** [ovh/the-bastion#54]
Hello,

I'm trying to use The Bastion through a iOS SSH client but I'm unable to make it work as the bastion needs some command on connection :
```
Bad or empty command
```

The app can send command on connect but I've tried many things without success :
```
-i
-- -i
ssh root@someserver
info
-i info
```

An option to allow to force session to be interactive if there's no command would be nice, even though I'm not sure it would work without option "-t" support on the given SSH client (doesn't work on my Terminal MacOS app).

The obvious goal is to have a better compatibility with any kind of SSH client app, not everyone uses Linux/MacOS/Unix terminal ;)

**Comment by speed47** [comment]
Interesting use case, this could be a configurable option indeed! (and possibly enabled by default for new installs).

The "send command" should work however, hmm. Which app are you using? To check if I can reproduce.

**Comment by snk33** [comment]
I'm using the excellent (and Made in France) WebSSH app. There's a command feature that allows to set a favorite command and send it on connection but it doesn't work :(

**Comment by speed47** [comment]
Ok, let me take a iThingy out of one of my drawers, charge it a bit, and test this app ;)

**Comment by isontheline** [comment]
Hello @snk33, Hello @speed47 👋

I'm the [WebSSH](https://apps.apple.com/fr/app/webssh/id497714887) developer 😊
Thank you @snk33 for issue https://github.com/isontheline/pro.webssh.net/issues/110

I will also check on my side if I can add support of "-t" switch inside WebSSH.

I think I will also use [the-bastion](https://github.com/ovh/the-bastion) in our office 👍

**Comment by snk33** [comment]
I’m glad to establish the link between 2 nice french projects :)

**Comment by speed47** [comment]
Hey @isontheline, nice to hear, thanks ;)
Don't hesitate if you have any questions!

**Comment by speed47** [comment]
@snk33 the v3.01.00 version has the option to drop the user into interactive mode if nothing is specified on the command-line.

As the implicit behavior of the `openssh` client is to imply `-t` when there is nothing specified on the command line (i.e. the user wants to have a remote shell and so they need a pseudotty), maybe that'll work out of the box with WebSSH. It depends whether the lib @isontheline uses also has this behavior or not ;)

**Comment by snk33** [comment]
I've updated our Bastion and it works as expected.

Good news, I've also managed to connect through WebSSH, it prompts "osh" as expect so I can type any command like "ssh user@host" and bastion works well.

Thanks a lot for this quick implementation ! I'll make some more tests but I guess any further adjustments will mostly be on WebSSH side (like send command on connection so allow direct connection to a server through the bastion).
